### ISSUE TITLE
Crash probably people hacking?

### ISSUE THREAD

**Issue by IvanTurgenev** [codehz/ElementZero#41]
docker linux essential mods no encryption
```
Version: 1.14.60.5 with element-0 (200419)
OS: Windows
Server start: 2020-05-04 23:02:17 Greenwich S
Dmp timestamp: 2020-05-05 03:50:23 Greenwich S
Upload Date: 2020-05-05 03:50:23 Greenwich S
Session ID: 475c26ba-60ff-4367-bd45-799398fa8cf0
Commit hash: 37aca878ca1c71648b7646e6758b1cb97dba79b1
Build id: 3998270
CrashReporter Key: ec7e7127-c0b2-30fe-94c2-3bec1fb4892f

Crash
I [SERVER] (CrashHelper::dumpCrashHandlerAppCrashLog:37)        at enum EventResult ScriptServerBlockEventListener::onBlockExploded[class BlockPos const &,class Block const &,class Actor *] (UnknownFile:?)                                
        at ?processEvent@?$EventCoordinator@VNetworkPacketEventListener@@@@IEAAXV?$function@$$A6A?AW4EventResult@@PEAVNetworkPacketEventListener@@@Z@std@@@Z (UnknownFile:?)                                                                 
        at void Explosion::finalizeExplosion[void] (UnknownFile:?)
        at void Level::explode[class BlockSource &,class Actor *,class Vec3 const &,float,bool,bool,float,bool] (UnknownFile:?)                                                                                                              
        at bool BedBlock::use[class Player &,class BlockPos const &]const  (UnknownFile:?)
        at class Block const & BlockPalette::getBlock[unsigned int const &]const  (UnknownFile:?)
        at <lambda_68877bb2d245f68df98aec72c86f713f>::operator[] (UnknownFile:?)
        at void Container::createTransactionContext[class A6AXAEAVContainer::function,int,class ItemStack const &,class ItemStack const &] (UnknownFile:?)                                                                                   
        at void PlayerInventoryProxy::createTransactionContext[class A6AXAEAVContainer::function,int,class ItemStack const &,class ItemStack const &] (UnknownFile:?)                                                                        
        at enum InventoryTransactionError ItemUseInventoryTransaction::handle[class Player &,bool]const  (UnknownFile:?)                                                                                                                     
        at void ServerNetworkHandler::handle[class NetworkIdentifier const &,class InventoryTransactionPacket const &] (UnknownFile:?)                                                                                                       
        at bool NetworkHandler::_sortAndPacketizeEvents[class NetworkHandler::Connection &,class std::chrono::time_point<struct std::chrono::steady_clock,class std::chrono::duration<__int64,struct std::ratio<1,1000000000> > >] (UnknownFile:?)
        at <lambda_b4722b03a5659a91d48c3c8a5a3c372a>::operator[] (UnknownFile:?)
        at void NetworkHandler::runEvents[bool] (UnknownFile:?)
        at bool Minecraft::update[void] (UnknownFile:?)

abnormal program termination
```

**Comment by Talark-Systems** [comment]
This looks like a script error:
~~~
EventResult ScriptServerBlockEventListener
~~~

~~~
at void Explosion::finalizeExplosion[void] (UnknownFile:?)
        at void Level::explode[class BlockSource &,class Actor *,class Vec3 const &,float,bool,bool,float,bool] (UnknownFile:?)
~~~

**Comment by IvanTurgenev** [comment]
i have no scripts

**Comment by Talark-Systems** [comment]
Has anything like this happened again? I also take back my previous comment, yes, this may have been caused by a hacked client, try enabling the anti-cheat mod. Also: do you have a log of the players that joined the server? That may help you find the hacker.

**Comment by IvanTurgenev** [comment]
Should probably leave it open, thought it might be bds bug depends how you look at it

**Comment by Bluebotlaboratories** [comment]
Why are you trying to type % in anyway?

**Comment by nrss** [comment]
> Why are you trying to type % in anyway?

I assume you meant to add this comment under my bug post. Not to be rude or anything, but why does it matter? It is still worth fixing, and I would imagine it would be relatively easy to fix.

**Comment by Bluebotlaboratories** [comment]
Not to be rude either, but the reason it doesn’t support % is probably more complicated than you think as this software, you must understand, is not server software, and it is an application what USES the official server software and adds mods as an added bonus, the link between the two softwares is probably very difficult and most likely the cause of your problem.

tl;dr: I’m not too sure but it might have something to do with the way this uses the official server software.

**Comment by Extollite** [comment]
You talk nonsense about stuff you don't have any idea.

**Comment by Bluebotlaboratories** [comment]
> You talk nonsense about stuff you don't have any idea.
> 
> 

Perhaps, but at least I tried, besides isn’t this a bit off-topic

**Comment by Extollite** [comment]
> Not to be rude either, but as a **programmer myself** it may have something to do with the way it interfaces with the official software. I’m not too sure about the inner-workings of this system.

You refer to yourself as programmer and C++ programmer and you don't understand what mods do. Please stop act as you are really helpful, but only spamming in issues.

**Comment by Bluebotlaboratories** [comment]
> > Not to be rude either, but as a **programmer myself** it may have something to do with the way it interfaces with the official software. I’m not too sure about the inner-workings of this system.
> 
> 
> 
> You refer to yourself as programmer and C++ programmer and you don't understand what mods do. Please stop act as you are really helpful, but only spamming in issues.

Elaborate by ‘you don’t know what mods do’

**Comment by Extollite** [comment]
For real man, you have member badge because you told that you will make wiki page, and now act as someone with knowledge about how things works. Do what you told will do and stop giving invaluable comments.

**Comment by Bluebotlaboratories** [comment]
Fine

**Comment by Extollite** [comment]
Do whatever you want till you don't say nonsense

**Comment by truaddict** [comment]
EZ and bds versions both were updated since the time of this report. Please reopen if bug is still relevant for you.
